mobile-app/app/screens/AppNavigator/screens/Loans/screens/VaultDetailScreen.tsx

```diff
@@ -0,0 +1,306 @@
+    },
+    {
+      iconName: 'tune',
+      iconType: 'MaterialIcons',
+      label: 'EDIT SCHEME',
+      disabled: true,
+      handleOnPress: () => {}
+    }
+  ]
+  const vaultChildTabs = [
+    {
+      id: TabKey.ActiveLoans,
+      label: 'Active loan',
+      disabled: false,
+      handleOnPress: onPress
+    },
+    {
+      id: TabKey.Details,
+      label: 'Details',
+      disabled: false,
+      handleOnPress: onPress
+    },
+    {
+      id: TabKey.Collaterals,
+      label: 'Collaterals',
+      disabled: false,
+      handleOnPress: onPress
+    },
+    {
+      id: TabKey.Auctions,
+      label: 'Auctions',
+      disabled: true,
+      handleOnPress: onPress
+    }
+  ]
+
+  return (
+    <ThemedScrollView
+      light={tailwind('bg-white border-gray-200')}
+      dark={tailwind('bg-gray-800 border-gray-800')}
+      style={tailwind('border-b')}
+    >
+      <View style={tailwind('p-4')}>
+        <VaultIdSection vaultId={vaultId} collaterals={currentVault.collaterals} />
+        <VaultCollateralTokenShare collaterals={currentVault.collaterals} />
+        <VaultInfoSection {...currentVault} />
+        <CollateralStatusMessage collateralRatio={currentVault.collateralRatio} />
+      </View>
+      <ThemedView
+        light={tailwind('border-gray-200')}
+        dark={tailwind('border-gray-700')}
+        style={tailwind('pb-4 border-b')}
+      >
+        <ScrollableButton buttons={vaultActionButtons} containerStyle={tailwind('pl-4')} />
+        <EmptyCollateralMessage collaterals={currentVault.collaterals} />
+      </ThemedView>
+      <Tabs tabSections={vaultChildTabs} activeTabKey={activeTab} />
+      <ThemedView
+        style={tailwind('h-full')}
+      >
+        {/* TODO: detail tab content */}
+      </ThemedView>
+    </ThemedScrollView>
+  )
+}
+
+function VaultIdSection (props: { vaultId: string, collaterals: Collateral[] }): JSX.Element {
+  return (
+    <ThemedView
+      light={tailwind('bg-white')}
+      dark={tailwind('bg-gray-800')}
+      style={tailwind('flex flex-row items-center')}
+    >
+      <ThemedView
+        light={tailwind('bg-gray-100')}
+        dark={tailwind('bg-gray-700')}
+        style={tailwind('w-8 h-8 rounded-full flex items-center justify-center mr-2')}
+      >
+        <ThemedIcon
+          iconType='MaterialIcons'
+          name='shield'
+          size={14}
+          light={tailwind('text-gray-600')}
+          dark={tailwind('text-gray-300')}
+        />
+      </ThemedView>
+      <View
+        style={tailwind('flex flex-1')}
+      >
+        <View
+          style={tailwind('flex flex-row mb-0.5')}
+        >
+          <ThemedText
+            light={tailwind('text-gray-400')}
+            dark={tailwind('text-gray-500')}
+            style={tailwind('text-xs mr-1.5')}
+          >
+            {translate('screens/VaultDetailScreen', 'Vault ID')}
+          </ThemedText>
+          <TouchableOpacity onPress={() => Clipboard.setString(props.vaultId)}>
```

Comment:
when we click on copy btn, it should have some visuals, currently when we click on it user dose not see that text is copied

Reply:
Asking for his opinion. But previously he did a toast design on `Receive screen` 
![image](https://user-images.githubusercontent.com/40191153/139827969-088753f5-2b10-41f7-9d8c-a477a4fc5b4a.png)

But maybe we can track this in a separate issue?

Reply:
Sure, Lets make a ticket for this, so that we wont forget about this

Reply:
👌https://github.com/DeFiCh/wallet/issues/1214

Reply:
I suggest we don't use copy here but just redirect it to DeFiScan. To be checked with @jtlntno-cake on a separate topic. FYI @kyleleow